packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Writers/CodeWriter.cs

```diff
@@ -225,7 +225,7 @@ public void WriteConstructor(ConstructorProvider ctor)
 
         internal void WriteXmlDocs(XmlDocProvider? docs)
         {
-            if (docs is null)
+            if (CodeModelPlugin.Instance.Configuration.DisableXmlDocs || docs is null)
```

Comment:
should we consider changing the implementation from "not writing" to not even construct them?
I used this way because this is the simplest way to implement this in autorest.csharp.
We do not have a centralized type or place to handle all the docs.
But I think it is possible in MGC to make it better.

Reply:
there isn't a good single place to do that in mgc either, this is the place that will for sure case all cases.